src/main/java/org/isf/opd/model/Opd.java

```diff
@@ -82,10 +82,10 @@
 
 	@NotNull
 	@Column(name="OPD_DATE_VIS")
-	private GregorianCalendar visitDate;
+	private Date visitDate;
         
 	@Column(name="OPD_DATE_NEXT_VIS")
-    private GregorianCalendar nextVisitDate;
+    private Date nextVisitDate;
```

Comment:
This is not correct, in the DB is a DATETIME and should be saved as that: it's an appointment, with date and time